manage.py

```diff
@@ -154,7 +155,14 @@ def run_server(self) -> None:
     def run_tests(self) -> None:
         """Prepare and run the test suite."""
         self.prepare_environment()
-        call_command(*sys.argv[1:])
+        # The way we run tests in CI, and whitenoise's behavior raise a warning
+        # since the static root folder does not need to exist
+        # Reference: https://github.com/evansd/whitenoise/issues/215
```

Comment:
Can you rephrase this? The clause before the comma seems incomplete.

Reply:
Fair enough, 74326128. Thanks for the review!